packages/commerce-sdk-react/src/hooks/createUseQuery.ts

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {DefaultError, UseQueryResult} from '@tanstack/react-query'
```

Comment:
I'm seeing a ts error here: 

```
Module '"@tanstack/react-query"' has no exported member 'DefaultError'.
```

Reply:
I'm not seeing the error in my machine. Perhaps double check that you have run `npm ci` to install the correct version of react-query.